apps/dashboard/package.json

```diff
@@ -16,12 +16,14 @@
     "@dnd-kit/sortable": "^8.0.0",
     "@dnd-kit/utilities": "^3.2.2",
     "@resvg/resvg-wasm": "^2.6.0",
+    "clsx": "^2.1.0",
     "next": "14.0.4",
     "react": "^18",
     "react-dom": "^18",
     "react-promise-suspense": "^0.3.4",
     "satori": "^0.10.11",
     "sonner": "^1.3.1",
+    "tailwind-merge": "^2.2.0",
```

Comment:
I don't think we'll *need* `tailwind-merge`, let's save 7kB and remove it

Reply:
I removed `tailwind-merge` for now, you're correct: it adds 7kb to the bundle and some runtime computation. imho this might be worth it in interactive apps where elements have a lot of different states (visible, selected, disabled etc.) whereas overriding using !important can become unpredictable. (the order is defined by the tailwind css output, not the className)

You can resolve this if you don't think it is worth it